packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -283,6 +283,10 @@ export default {
             (!this.address || !this.address.line1);
         },
 
+        shouldLoadCustomerNameFromClaims () {
+            return this.isLoggedIn && (!this.customer.firstName && !this.customer.lastName);
```

Comment:
I'm not sure if it could happen in Production, but what would happen if only the `firstName` is set but not the `lastName`, or vice-versa?

Reply:
A good point, but should we override what we get from the API in this case? So just change it to `||`

Reply:
Not sure. Maybe David M knows?